ui/app/templates/components/proxy-tag.hbs

```diff
@@ -0,0 +1 @@
+<span class="tooltip" role="tooltip" aria-label="Consul Connect proxy task">Proxy</span>
```

Comment:
Putting the tooltip on the text instead of the outer tag that has the padding results in the tooltip being rendered lower than I'd expect.

Reply:
ah okay, I’ve merged the elements, thanks.